test/spec/ol/interaction/modify.test.js

```diff
@@ -140,7 +140,7 @@ describe('ol.interaction.Modify', function() {
     // make sure we get change events to events array
     expect(events.length > 2).to.be(true);
     // middle events should be feature modification events
-    for (var i = 1; i < events.length - 2; i++) {
+    for (var i = 1; i < events.length - 1; i++) {
```

Comment:
If `events.length = 3`, `events = [modifystart, change, modifyend]`, the loop is only on 1, which is `change`. I don't think there is anything wrong